generators/app/templates/README.md

```diff
@@ -10,88 +14,100 @@ This README is intended to help get you started. Definitely update and improve t
 
 ### Running <%= botName %> Locally
 
-You can test your hubot by running the following.
+You can test your hubot by running the following, however some plugins will not
+behave as expected unless their specific environment variables have been set.
```

Comment:
I've added a configuration section, with how I go about this. It's not ideal and if there's a better way you can think of I'm all for it.